.gitignore

```diff
@@ -7,3 +7,9 @@ local.properties
 google-services.json
 /build.gradle
 _artifacts
+# VS code/extension files
+.project
+.classpath
+org.eclipse.buildship.core.prefs
+bin/
+.vscode/
```

Comment:
Out of curiosity, are you using VS Code instead of / in addition to Android Studio?  Does it work well?  I love VS Code in general, but haven't tried to use it with Android development.

That said...  since it is pretty uncommon to use VS Code (I think?) for Android and some of these directories are a bit surprising ("bin" especially) and could _potentially_ bite somebody someday...  I wonder if we should avoid adding them to the repo .gitignore.  You could consider using a global gitignore file for now: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Maybe I'm being overly cautious though.  Maybe @ashwinraghav or @vkryachko can chime in on behalf of the android core team.

Reply:
I was just using the git management there. Still using Android Studio for most of developing.

Changed to use global gitignore as you suggested.